src/client/pages/RetroCertsLandingPage/index.test.js

```diff
@@ -1,11 +1,57 @@
 import renderNonTransContent from "../../test-helpers/renderNonTransContent";
 import Component from "./index";
+import auth from "../../../data/auth";
 
 describe("<RetroCertsLandingPage />", () => {
-  it("renders the retro certs landing page (after auth)", async () => {
+  it("no authToken or user data", async () => {
+    sessionStorage.removeItem(auth.AUTHTOKEN);
     const wrapper = renderNonTransContent(Component, "RetroCertsLandingPage", {
       userData: {},
-      setUserData: () => {}
+      setUserData: () => {},
+    });
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("re-fetch data with authToken", async () => {
+    sessionStorage.setItem(auth.AUTHTOKEN, 'TEST_TOKEN');
+    // fetch isn't part of nodejs. If we pull in node-fetch, we
```

Comment:
interesting! side note, we mock fetch on the browser side for ie11 with whatwg-fetch :P

Reply:
Yeah, we'll probably need node-fetch for the validation of the reCAPTCHA token.